lib/app/modules/login/views/three_view.dart

```diff
@@ -21,90 +23,80 @@ class LoginView extends GetView<ThreeController> {
     return Scaffold(
         body: SingleChildScrollView(
       child: SafeArea(
-        child: Padding(
-          padding: EdgeInsets.all(Constants.defaultPadding.sp * 2),
-          child: Column(
-            children: [
-              LargeText(
-                'Login',
-              ),
-              SizedBox(
-                height: 10.h,
-              ),
-              NormalText('Add you details to log in'),
-              SizedBox(height: 20.h),
-              CustomTextField(
-                label: 'Email or Username',
-              ),
-              SizedBox(
-                height: 20.h,
-              ),
-              CustomTextField(
-                label: 'Password',
-              ),
-              SizedBox(
-                height: 20.h,
-              ),
-              CustomTextButton(
-                  label: 'Login',
-                  textColor: Theme.of(context).scaffoldBackgroundColor,
-                  onPressed: () {
-                    Get.toNamed('/onboarding');
-                  }),
-              SizedBox(
-                height: 16.h,
-              ),
-              GestureDetector(
-                  onTap: () {
-                    Get.to(() => ResetPasswordView());
-                  },
-                  child: NormalText('Forgot your password')),
-              SizedBox(
-                height: 20.h,
-              ),
-              NormalText('or Login With'),
-              SizedBox(
-                height: 20.h,
-              ),
-              CustomTextButton(
-                icon: FontAwesomeIcons.facebookF,
-                label: 'Facebook',
-                onPressed: () {},
-                btnColor: Colors.blue,
-              ),
-              SizedBox(
-                height: 20.h,
-              ),
-              CustomTextButton(
-                icon: FontAwesomeIcons.googlePlusG,
-                label: 'Login with Google',
-                onPressed: () {},
-                btnColor: Colors.red,
-              ),
-              SizedBox(
-                height: 20.h,
-              ),
-              RichText(
-                  text: TextSpan(
-                      text: "Don't have an Account? ",
-                      style: Theme.of(context)
-                          .textTheme
-                          .bodyText1!
-                          .copyWith(color: Colors.grey),
-                      children: [
-                    WidgetSpan(
-                        child: InkWell(
-                      onTap: () => Get.to(() => SignupView()),
-                      child: Text(
-                        'Sign up',
+        child: ResponsiveLayout(
+          mobile: Container(
+            height: ScreenUtil.defaultSize.height,
```

Comment:
you do not have to give height to container as the column itself takes height it needs